oracle/price-feeder/oracle/client/tx.go

```diff
@@ -25,15 +25,15 @@ func BroadcastTx(clientCtx client.Context, txf tx.Factory, msgs ...sdk.Msg) (*sd
 	// }
 
 	// txf = txf.WithGas(adjusted)
+	txf = txf.WithGas(0)
+	// txf = txf.WithFees("0usei")
```

Comment:
Whats the difference between WithFees vs SetFeeAmount?

Reply:
WithFees is on the factory level, and SetFeeAmount modifies the unsigned tx as a one off adjustment